### PR TITLE
New package: CyberSift.TutelaWindowsAgent version 1.5.0

### PR DESCRIPTION
**Pull request by dvas0004**
Checklist for Pull Requests
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Is there a linked Issue?

Manifests
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/doc/Authoring.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.6 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.6.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/156793)

### CODE REVIEW

**Comment by stephengillie** [comment]
Automatic Validation ended with:

> Installation failed with exit code -1978335187
> 2024-06-06 08:42:54.635 [FAIL] Installer failed security check. Url: http://repo.cybersift.io/cs-vas/tutela.msi Result: 0x80004005

Automated error analysis suggests -1978335187 may mean Installer failed security check. Another script should retry this every 18 hours. 

(Automated response - build 883.)

File: manifests/c/CyberSift/TutelaWindowsAgent/1.5.0/CyberSift.TutelaWindowsAgent.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.6.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.6.0.schema.json
+
+PackageIdentifier: CyberSift.TutelaWindowsAgent
+PackageVersion: 1.5.0
+PackageLocale: en-US
+Publisher: CyberSift
+PackageName: Tutela Windows Agent
+License: Commercial
+ShortDescription: Tutela Windows Agent
```
Comment (Trenly) [inline manifests/c/CyberSift/TutelaWindowsAgent/1.5.0/CyberSift.TutelaWindowsAgent.locale.en-US.yaml:10]: Can you please provide a better description of what this software is or what it is used for?

**Review by Trenly — CHANGES_REQUESTED** [review]

File: manifests/c/CyberSift/TutelaWindowsAgent/1.5.0/CyberSift.TutelaWindowsAgent.locale.en-US.yaml (hunk 1 above)
Comment (dvas0004) [reply manifests/c/CyberSift/TutelaWindowsAgent/1.5.0/CyberSift.TutelaWindowsAgent.locale.en-US.yaml]: Yes sure!
Updated short description with: `Tutela Windows Agent. This agent enumerates software running on a workstation to check for applicable CVEs. It also runs several CIS benchamrk checks. (https://www.cybersift.com/tutela)`

**Comment by stephengillie** [comment]
Automatic Validation ended with:

> Installation failed with exit code -1978335226
> 2024-06-06 23:52:40.262 [CLI ] ShellExecute installer failed: 1603
> WixQuietExec:  Error 0x800700d8: Command failed to execute.
> WixQuietExec:  Error 0x800700d8: QuietExec Failed
> WixQuietExec:  Error 0x800700d8: Failed in ExecCommon method
> CustomAction CustomInstallExec0 returned actual error code 1603 (note this may not be 100% accurate if translation happened inside sandbox)
> Property(S): ErrorDialog = ErrorDlg
> MSI (s) (B0:68) [23:52:40:070]: Product: Tutela Windows Agent -- Installation failed.

Automated error analysis suggests -1978335226 may mean Install successful but system restart may be required.

Automated error analysis suggests 0x800700d8 may mean This version of this application is not compatible with the version of Windows you're running. Check your computer's system information and then contact the software publisher. (Should this have `Architecture: x64`?)

Automated error analysis suggests 1603 may mean Install successful but system restart may be required. Should this be add as an `InstallerSuccessCode` or similar?

(Automated response - build 883.)

**Comment by stephengillie** [comment]
Manual Validation ended with:
> 
> 2024-06-07 08:30:38.815 [CLI ] Package hash verification failed. SHA256 in manifest [1d42750a0dd26fe9672f504d598a54caa692c70fed1a42d4e6ddce251302075c] does not match download [01efe25b1968ed68c160d256c7b1c55e7524978bf24c279c23ac3fab25ea8e2a]

(Automated response - build 883.)

**Comment by dvas0004** [comment]
@Trenly @stephengillie  checks finally passing... thank you so much for your helpful advice!! 🙂

**Comment by stephengillie** [comment]
Hi @dvas0004,

I'm happy to help. 😄 

Verify:
image
- `updater.exe` has an error - is it supposed to be run from command line? 

image
- `ETWReader.exe` needs `Microsoft.VCRedist.2015+.x64` (or `x86`) - should it be added as a dependency here? 

image

**Comment by dvas0004** [comment]
Submitting new manifest to address the following:

> updater.exe has an error - is it supposed to be run from command line?

It should be able to run yes:
image

From the error it seems there was some issue to connect to the windows service manager. I've updated `updater.exe`  to handle errors here more gracefully

> ETWReader.exe needs Microsoft.VCRedist.2015+.x64 (or x86) - should it be added as a dependency here?

Added, thank you!

**Comment by stephengillie** [comment]
Thanks for updating the updater and the manifest. All seems to be working as expected: 
image

**Review by stephengillie — APPROVED** [review]

**Pull request merged**